storage/migrations/13_related_transactions_method_denorm.up.sql

```diff
@@ -4,13 +4,52 @@ BEGIN;
 ALTER TABLE chain.accounts_related_transactions
 	ADD COLUMN method TEXT;
 
-UPDATE chain.accounts_related_transactions AS art
-	SET method = t.method
-	FROM chain.transactions AS t
-	WHERE art.tx_block = t.block AND art.tx_index = t.tx_index;
+DO $$
+DECLARE
+    block_batch_size INT := 50000;
+    start_block INT;
+    end_block   INT;
+    max_block   INT;
+    affected_rows INT;
+BEGIN
```

Comment:
some of this stuff seems to be in the wrong commit?

Reply:
Fixed!